build/yaml/samples/nsx_v1alpha1_ipaddressallocation.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: nsx.vmware.com/v1alpha1
+kind: IPAddressAllocation
+metadata:
+  name: guestcluster-workers-a
+  namespace: sc-a
+spec:
+  ip_address_block_visibility: Private
+  allocation_size: 26
```

Comment:
does allocation_size need to be power of 2?

Reply:
No need, nsx will decide the cidr based on this size.